classes/patreon_metabox.php

```diff
@@ -49,14 +46,53 @@ function patreon_plugin_meta_box( $object, $box ) {
 			$readonly = " readonly";
 		}
 
-			wp_nonce_field( basename( __FILE__ ), 'patreon_metabox_nonce' ); ?>
+			wp_nonce_field( basename( __FILE__ ), 'patreon_metabox_nonce' ); 
+			
+		?>
 		<p>
 			<label for="patreon-level"><?php _e( $label, '1' ); ?></label>
 			<br><br>
 			<strong>&#36; </strong><input type="text" id="patreon-level" name="patreon-level" value="<?php echo get_post_meta( $object->ID, 'patreon-level', true ); ?>" <?php echo $readonly ?>>
 		</p>
 
 		<?php
+		
+		if(get_option('patreon-can-use-api-v2',false)=='yes') {
+			
+			$label = 'Active patrons only (optional) - if on, only patrons active at and before this post\'s publishing date will be able to see this content )';
+			$readonly = '';
+			
+			if(!get_option('patreon-creator-id', false)) {
+				$label = 'Post locking won\'t work without Creator ID. Please confirm you have it <a href="'.admin_url("?page=patreon-plugin").'">here</a>';
+				$readonly = " readonly";
+			}
+
+			?>
+			<p>
+				<label for="patreon-active-patrons-only"><?php _e( $label, '1' ); ?></label>
+				<br><br>
+				<input type="checkbox" name="patreon-active-patrons-only" value="1" <?php checked( get_post_meta( $object->ID, 'patreon-active-patrons-only', true ),true,true ); ?> <?php echo $readonly ?> /> Yes
+			</p>
+
+			<?php
+			
+			$label = 'Total patronage (optional) - any patron above total lifetime patronage $ value entered below can see this content';
+			$readonly = '';
+			
+			if(!get_option('patreon-creator-id', false)) {
+				$label = 'Post locking won\'t work without Creator ID. Please confirm you have it <a href="'.admin_url("?page=patreon-plugin").'">here</a>';
+				$readonly = " readonly";
+			}
+
+			?>
+			<p>
+				<label for="patreon-total-patronage-level"><?php _e( $label, '1' ); ?></label>
+				<br><br>
+				<strong>&#36; </strong><input type="text" id="patreon-total-patronage-level" name="patreon-total-patronage-level" value="<?php echo get_post_meta( $object->ID, 'patreon-total-patronage-level', true ); ?>" <?php echo $readonly ?>>

```

Comment:
I don't love this html formatting, have we made a call on how we format inline html strings like this?

Reply:
Commenting here to address earlier comments:

I believe ascii &#36; was used in place of $ to make it compatible with any browser in any language. I think it is good for this purpose. Even though the plugin is not multi-language yet.